templates/workers.yaml.j2

```diff
@@ -39,7 +39,7 @@ spec:
       - name: "celery-worker"
         image: "{{ k8s_worker_container["image"] }}:{{ k8s_worker_container["tag"] }}"
         imagePullPolicy: "{{ k8s_worker_container["image_pull_policy"] }}"
-        args: ["celery", "--app={{ k8s_worker_container["celery_app"] }}", "worker", "--workdir=/code", "--loglevel=info"]
+        command: {{ k8s_worker_command }}
```

Comment:
Should this be using ``worker_command`` instead of ``k8s_worker_command``?

Reply:
Hmm, i'd think that it would be either `k8s_worker_command` or `k8s_worker_container["worker_command"]`, but I guess I don't understand the levels of abstraction that are going on here.

But otherwise these changes look good to me, so if @dpoirier or @JeremyGibson want to agree on a term here, I can make the change here (and in the beat template, if needed), and then I can merge and release this.

Reply:
I think I must have meant I thought the K8S template should be using `k8s_worker_container["worker_command"]`. That seems consistent with how the other variables are being used.  

Though honestly I don't know why the variables for the worker all seem to be gathered into `k8s_worker_container`. @copelco @JeremyGibson can you help?

Reply:
I think it was modeled after ``k8s_web_containers``. I think ``k8s_worker_container["worker_command"]`` is correct.